### ISSUE TITLE
Custom Pluralization

### ISSUE THREAD

**Issue by elistia** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#605]
Hi,

When I run the generator against my database some of the default pluralization does not work correctly. For instance the table 'SupportStatus' creates a class 'SupportStatu'.

In v2 of the generator (running in a Framework app) I could do the following:

```c#
// If pluralisation does not do the right thing, override it here by adding in a custom entry.
Inflector.PluralizationService = new EnglishPluralizationService(new[]
{
    new CustomPluralizationEntry("Response", "Responses"),
    new CustomPluralizationEntry("SupportStatus", "SupportStatus")
});
```

This worked perfectly. However, in v3 (.net core this time) I get the following error when the above code is used in the Database.tt:

`Compiling transformation: 'GeneratedTextTransformation.Inflector' does not contain a definition for 'PluralizationService'`

Is there a different way of doing this now, or is this as simple as needing to install the appropriate NuGet package to make it work?

Thanks,

**Comment by sjh37** [comment]
Hi. For v3, I changed `PluralizationService` to be `PluralisationService`. Try:
```c#
Inflector.PluralisationService = new EnglishPluralizationService(new[]
{
    new CustomPluralizationEntry("Response", "Responses"),
    new CustomPluralizationEntry("SupportStatus", "SupportStatus")
});
```

Sorry for any confusion.

**Comment by elistia** [comment]
Thanks Simon - I could have stared at that for hours and never spotted the difference!

Just as an observation - when I added in the code as above the generator created the new classes with the correct names, but the classes with the old spellings remained as well. I seem to remember last time I did this with v2 the old ones were removed?

This isn't a problem in the slightest - can easily delete the incorrect ones, but thought I would point it out in case anyone else is looking at this in the future.

Cheers.
